package.json

```diff
@@ -24,10 +24,10 @@
     "node": ">=6.9"
   },
   "scripts": {
-    "start": "npm run init:generated && npm run fetch && node ./antwar.bootstrap.js develop",
+    "start": "yarn init:generated && yarn fetch && node ./antwar.bootstrap.js develop",
```

Comment:
This is a problem for new contributors, Yarn is not necessary to run build and run this repository.
Make `yarn` mandatory creates an unnecessary new barrier for new contributors.

Yes, we use `yarn.lock` for the CI, you changed CI to use Yarn, but new contributors shouldn't have in it order to contribute to the documentation.

Reply:
Thanks for the review.

I get your concern, but currently, if you follow instructions from `CONTRIBUTING.md` step-by-step, running `npm install` is creating `package-lock.json` file which can also create confusion, for example, contributors might accidentally commit it and having two lock files it's not something we want to encourage, right?

Perhaps we can at least add `package-lock.json` and state in the guides that usage of yarn is more appreciated but not mandatory? What do you think?

Reply:
Please add `package-lock.json` to `.gitignore`, Same as webpack repo.